spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/properties/RateLimitProperties.java

```diff
@@ -315,12 +336,35 @@ public void setOrigins(List<String> origins) {
             this.origins = origins;
         }
 
-        public int getResponseStatusCode() {
-            return responseStatusCode;
+    }
+
+    public static class Location {
+
+        /**
+         * List of origins that will have the request denied.
+         */
+        private List<String> deny = new ArrayList<>();
```

Comment:
You could consider replacing the `List<String>` with a `Set<String>`, this will allow you to do a `Set.contains()` instead of a `List.stream().anyMatch(....)`